server.py

```diff
@@ -82,14 +117,51 @@ def do_get_json_sequence():
 
 @app.route('/healthcheck', methods=['GET'])
 def healthcheck():
+    try:
+        conn = db.engine.connect()
+        test_sql(conn)
+    except SQLAlchemyError:
+        logger.fatal("Failed to connect to database")
```

Comment:
Is it possible to log out the headline of the error as a KV pair? Or the full traceback if we can't just get a portion of it.